docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,347 @@
+    block's student view. The HTML can include CSS and Javascript code. This
+    field can be used in combination with the ``student_view_multi_device``
+    field to decide whether to display this content to the user. This URL can
+    be used as a fallback if the ``student_view_data`` for this block type is
+    not supported by the client or by the block.
+
+  * student_view_multi_device: (boolean) Whether or not the block's rendering
+    obtained via block_url has support for multiple devices. Returned only if
+    "student_view_multi_device" is included in the "requested_fields"
+    parameter.
+
+  * lms_web_url: (string) The URL to the navigational container of the xBlock
+    on the web LMS. This URL can be used as a further fallback if the
+    student_view_url and the student_view_data fields are not supported.
+
+  * lti_url: The block URL for an LTI consumer.
+
+
+================
+Return Values
+================
+
+* 200 on success with above fields.
+
+* 400: Bad Request for the following error conditions.
+
+  * If ``course_id`` is not provided
+  * If the provided ``course_id`` is invalid
+  * If ``username`` is not provided in cases other than ``all_blocks``
+    requested.
+
+* 404: If the specified block is not found.
+
+
+============================================================================
+Example Response Showing a List of Course Blocks in a Specified Course
+============================================================================
+
+.. code-block:: json
+
+ {
+   "root": "i4x://edX/DemoX/course/Demo_Course",
+   "blocks": {
+      "i4x://edX/DemoX/course/Demo_Course": {
+        "display_name": "edX Demonstration Course",
+        "graded": false,
+        "student_view_url": "https://courses.edx.org/xblock/i4x://edX/DemoX/
+         course/Demo_Course",
+        "student_view_multi_device": false,
+        "lms_web_url": "https://courses.edx.org/courses/edX/DemoX/Demo_Course/
+          jump_to/i4x://edX/DemoX/course/Demo_Course",
+        "type": "course",
+        "id": "i4x://edX/DemoX/course/Demo_Course"
+        }
+    }
+ }
+
+
+.. _Get a list of the course blocks in a block tree:
+
+*********************************************
+Get a List of Course Blocks in a Block Tree
+*********************************************
+
+The endpoint to get a list of course blocks in a specified block tree is
+``/api/courses/v1/blocks/<usage_id>/``.
+
+=====================
+Use Case
+=====================
+
+Get a list of course blocks in a specified block tree. Response results depend
+on the requesting user's access level.
+
+=====================
+Example Request
+=====================
+
+GET /api/courses/v1/blocks/<usage_id>/
+
+GET /api/courses/v1/blocks/i4x%3A%2F%2FedX%2FDemoX%2Fvertical
+%2F2152d4a4aadc4cb0af5256394a3d1fc7?all_blocks=true
+
+
+=====================
+Parameters
+=====================
+
+This view accepts the following parameters.
```

Comment:
I think we should remove all references to the view. Maybe:

The ``/api/courses/v1/blocks/{usage_id}/`` resource accepts the following query parameters.